doc/overview/datasets_index.rst

```diff
@@ -119,9 +119,6 @@ Resting state
 
 Details can be found at the Brainstorm `resting state dataset tutorial`_.
 
-.. topic:: Examples
-
-    * :ref:`ex-envelope-correlation`
```

Comment:
Ideally instead of removing this, we would use intersphinx and be able to do
```
* :ref:`mne-connectivity:ex-envelope-correlation`
```
or similar...